### PR TITLE
Update error copy, check a11y

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Update the copy on the 403 page to match 404. Get rid of the double `<main>` and do some a11y tests

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All pages have visual tests via cypress + percy

### How can I view this in a review app?
Go to `/403`, `/404`, `/500`

### CODE REVIEW

File: app/views/errors/not_found.html.erb [hunk 1]
```diff
@@ -7,4 +6,3 @@
       <p class="govuk-body">If the web address is correct or you selected a link or button, contact: <%= render MailToSupportComponent.new %></p>
     </div>
   </div>
-</main>
```
Comment (kicferk1) [inline app/views/errors/not_found.html.erb:10]: Do we want to un-indent the html inside?

File: app/views/errors/forbidden.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
-<h1>Forbidden</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Page not found</h1>
+    <p class="govuk-body">If you typed the web address, check it is correct.</p>
+    <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
+    <p class="govuk-body">If the web address is correct or you selected a link or button,
+      contact: <%= render MailToSupportComponent.new %></p>
```
Comment (kicferk1) [inline app/views/errors/forbidden.html.erb:7]: That's a funny message for a 403 page, but I guess we don't want do let people map our site by giving them information that a page is 'real but not allowed'.

**Review by kicferk1 — APPROVED** [review]

File: app/views/errors/forbidden.html.erb (hunk 2 above)
Comment (ethanmills) [reply app/views/errors/forbidden.html.erb:7]: Alaric said that's a common pattern for Gov, that to a user perspective there's no difference between "you can't see this" and "this doesn't exist"

Comment (kicferk1) [reply app/views/errors/forbidden.html.erb:7]: Sure there is a difference, if it doesn't exist, you think that maybe you got a dead url. But if it exists and you can't see it, well, you might need to ask for some permissions. So for a determined user, different messages point them to different first course of action.

Whether having incorrect permissions is a common thing in government is a common problem or not, eh, who knows.

**Pull request merged**